i18n-util/src/UrlUtil.js

```diff
@@ -0,0 +1,28 @@
+
+const getAliasUrl = (location) => {
+  let cshid = null;
+  if (location.hash && location.hash.length > 0) {
+    cshid = location.hash.split('#')[1];
```

Comment:
nit: String processing is quicker!

```suggestion
    cshid = location.hash.substring(1);
```